src/einsteinpy/symbolic/tensor.py

```diff
@@ -307,3 +307,14 @@ def symbols(self):
         
         """
         return self.syms
+
+    def function(self, *args):
```

Comment:
`function` is not a good name. Something's wrong it, don't know what!
Some other suggestion? @OVSofia @shreyasbapat

Reply:
maybe `lambdify_narray` ?